packages/app/.travis.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Can we remove this?

Reply:
the entire file?

Reply:
Yes, we use the top level .travis.yml file.

Reply:
we didn't remove navi-data one.  Also, do we really want to remove this on every ember-upgrade?